tests/lido/ethereum/lido_ethereum_accounting_test.sql

```diff
@@ -0,0 +1,17 @@
+with  tests as (
+select  'Check Lido deposits' as test_name, 
+case when amount_staked = 224000000000000000000 then true else false end as success
```

Comment:
Hi @gregshestakovlido, the test fails for `Check Lido deposits`.
https://github.com/duneanalytics/spellbook/actions/runs/4696020625/jobs/8325724396?pr=3128#step:10:19

The amount_staked seems to be `32000000000000000000`
https://dune.com/queries/2367842?d=1

Please verify which result is correct.

Reply:
Thank you, fixed